osquery/sql/tests/sql_tests.cpp

```diff
@@ -154,4 +154,11 @@ TEST_F(SQLTests, test_sql_sha256) {
   EXPECT_EQ(d[0]["test"],
             "9f86d081884c7d659a2feaa0c55ad015a3bf4f1b2b0b822cd15d6c15b0f00a08");
 }
+
+TEST_F(SQLTests, test_sql_regex_replace) {
```

Comment:
C++ vs UTF8... grrrr.

So Vincent said the test fails when using UTF8... That's kind of sad isn't it? This means every attempt to anonymize a string containing UTF8 characters would fail? What would happen to the daemon (I hope it doesn't crash)? Would the anonymization fails (and we see the username in the logs)?